todo/src/components/TodoItem.js

```diff
@@ -1,17 +1,50 @@
 import React, { useState } from "react"
 
 function TodoItem(props) {
-	const [checked, setChecked] = useState(false)
 	return (
-		<div>
+		<div
+			style={{
+				display: "flex",
+				border: `1px solid ${props.darkModeEnabled ? "#f1f1f1" : "#333333"}`,
+				marginBottom: "1.5vh",
+				borderRadius: "5px",
+			}}
+		>
 			<label>
 				<input
+					style={{
+						borderRadius: "5px",
+					}}
 					type="checkbox"
-					defaultChecked={checked}
-					onChange={() => setChecked(!checked)}
+					defaultChecked={props.cb}
+					onChange={() => {
+						props.onChange(props.id)
+					}}
 				/>
 			</label>
-			{props.index + 1 + " " + props.content}
+			<label

```

Comment:
Why did you create a new `label`?

Reply:
That it would have a different styling when you click on the checkbox it will have a line through

Reply:
Try using the label you already created and tell me what the difference was (use the style you used in the second label).